egs2/TEMPLATE/enh_asr1/scripts/utils/upload_models_to_hub.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```

Comment:
Better to replace with a symbolic link to [egs2/TEMPLATE/asr1/scripts/utils/upload_models_to_hub.sh](https://github.com/espnet/espnet/blob/master/egs2/TEMPLATE/asr1/scripts/utils/upload_models_to_hub.sh).